pycontrails/physics/jet.py

```diff
@@ -8,15 +8,20 @@
 from __future__ import annotations
 
 import logging
+import pathlib
 
 import numpy as np
+import pandas as pd
 import numpy.typing as npt
 
 from pycontrails.core import flight
 from pycontrails.physics import constants, units
 from pycontrails.utils.types import ArrayOrFloat, ArrayScalarLike
 
 logger = logging.getLogger(__name__)
+_path_to_static = pathlib.Path(__file__).parent / "static"
+PLF_PATH = _path_to_static / "iata-passenger-load-factors-20241115.csv"
+CLF_PATH = _path_to_static / "iata-cargo-load-factors-20241115.csv"
```

Comment:
Can you confirm that both of these files are fine to share publicly?

Reply:
Yes, I have created these csv’s from reports that are publicly available and published by IATA